examples/stocks/test_driver/scroll_perf_test.dart

```diff
@@ -6,7 +6,7 @@ import 'dart:async';
 
 import 'package:flutter_driver/flutter_driver.dart';
 // ignore: deprecated_member_use
-import 'package:test_api/test_api.dart' hide TypeMatcher, isInstanceOf;
+import 'package:test/test.dart' hide TypeMatcher, isInstanceOf;
```

Comment:
@zanderso this is to account for: https://github.com/flutter/flutter/issues/45624